Doc/library/intro.rst

```diff
@@ -64,3 +64,50 @@ Notes on availability
   libc version, then both conditions must hold. For example a feature with note
   *Availability: Linux >= 3.17 with glibc >= 2.27* requires both Linux 3.17 or
   newer and glibc 2.27 or newer.
+
+.. _wasm-availability:
+
+Notes on WebAssembly platforms
+==============================
+
+The WebAssembly platforms ``wasm32-emscripten`` and ``wasm32-wasi`` provide a
+subset of POSIX APIs. WASM runtimes and browser are sandboxed and have
+limited access to the host and external resources. Any standard library module
+that uses processes, threading, networking (:mod:`socket`), signals, or
```

Comment:
Is there a reason why only the `socket` module is specified and not other modules?

Reply:
Which other modules do you expect? All networking modules are based on the socket module.

Reply:
I just found it a bit odd that there are no mentions about specific modules for processes (e.g. `subprocess`, `multiprocessing`) or threading, and one single example for networking.

Reply:
Makes sense. I have removed the reference to the socket module from the sentence. Initially I wanted to include examples for each missing feature, but then it turned out to make the sense much harder to read.